Plugins/Flow.Launcher.Plugin.PluginsManager/Main.cs

```diff
@@ -50,9 +51,9 @@ public async Task<List<Result>> QueryAsync(Query query, CancellationToken token)
             return query.FirstSearch.ToLower() switch
             {
                 //search could be url, no need ToLower() when passed in
-                Settings.InstallCommand => await pluginManager.RequestInstallOrUpdate(query.SecondToEndSearch, token),
+                Settings.InstallCommand => await pluginManager.RequestInstallOrUpdate(query.SecondToEndSearch, token, query.IsReQuery),
                 Settings.UninstallCommand => pluginManager.RequestUninstall(query.SecondToEndSearch),
-                Settings.UpdateCommand => await pluginManager.RequestUpdateAsync(query.SecondToEndSearch, token),
+                Settings.UpdateCommand => await pluginManager.RequestUpdateAsync(query.SecondToEndSearch, token, query.IsReQuery),
```

Comment:
why requery means use the primary url?

Reply:
I wanted to add a way to fetch fresh data (not cached data from a CDN) and this seemed a reasonable solution, if you think of `Ctrl+R` as a refresh mechanism.

This is mostly a temporary hack for #2048 honestly, I think we can rethink this as part of #2178

Reply:
I see. I previously thought `Last()` refers to cdn.